src/argilla/server/alembic/versions/bda6fe24314e_create_vectors_table.py

```diff
@@ -34,14 +34,17 @@ def upgrade() -> None:
     op.create_table(
         "vectors",
         sa.Column("value", sa.JSON(), nullable=False),
-        sa.Column("vector_settings_id", sa.Uuid(), nullable=False),
+        sa.Column("dataset_id", sa.Uuid(), nullable=False),
```

Comment:
I already commented this to Gabriel. Not sure why we are duplicating the association to the dataset when vector is already associated to the dataset through the record, maybe I'm missing something.

Ted Codd would not have approved this change. 😝

Reply:
We have 2 kinds of entities related to vectors, the vector configuration/settings (at the dataset level) and the vector value itself (at the record level). Let's see the `vector_settings` as the vector schema and `vectors` as the value for those for each record